hosts/motherlode/configuration.nix

```diff
@@ -0,0 +1,28 @@
+{ config, pkgs, ... }:
+let
+  variables = import ../../common/variables.nix;
+in {
+  imports = [
+    ./hardware-configuration.nix
+    ../../common/sysconfig.nix
+    ../../services/ssh.nix
```

Comment:
thinking aloud should `services/ssh` be moved into `common/sysconfig`

Reply:
Yeah, probably. I'll change it after mail merge.